SwiftCompilerSources/Sources/SIL/Instruction.swift

```diff
@@ -47,6 +47,11 @@ public class Instruction : CustomStringConvertible, Hashable {
     return OperandArray(base: operands.base, count: operands.count)
   }
 
+  final public var realOperands: OperandArray {
+    let operands = bridged.getRealOperands()
```

Comment:
Actually, I really don't want to do this because we shouldn't need two bridging calls every time we ask for the defined (non-typedependent) operands, which should really be the common case.

Reply:
Done. Unfortunately, either way, we have two calls into SILInstruction.cpp. As long as SILBridging.h code can be inlined into Swift code the choice it doesn't seem to matter much.